USSF-ORBIT/ussf-portal-client
# fix: Fix login success redirect status

# SC-326

## Proposed changes

Explicitly use a 302 redirect on a successful login. The root issue of this bug was when [updating NextJS from 12.0.10 -> 12.1.0](https://github.com/USSF-ORBIT/ussf-portal-client/commit/8867fac87be5570b87341cc97dfdec13e3801bcd), the default redirect status [was changed from 302 to 307](https://github.com/vercel/next.js/pull/33505/files).

## Reviewer notes

To test:
- Log in to the portal and verify you do not see the 405 error page
- Join the beta, log out, then log back in and verify you still do not see the 405 error page
- Verify the above in Chrome, Safari, Firefox

## Setup

This should be tested on the dev AWS environment.

---

## Code review steps

### As the original developer, I have


### As code reviewer(s), I have


### As a designer reviewer, I have
